distribution/lib/Standard/Table/0.0.0-dev/src/Io/Csv.enso

```diff
@@ -64,19 +64,18 @@ from_csv : File.File | Text -> Boolean -> Text -> Table ! Parse_Error
 from_csv csv has_header=True prefix='C' =
```

Comment:
Could this file and Delimited_Reader be merged?

Reply:
I would avoid it - `Csv` is the old module that should be deprecated and removed once `Delimited` is on-par with its functionality.

So making them separate makes it easier to see what is going to be removed. We keep it only because our Table tests _heavily_ rely on this mechanism and `Delimited` is not yet fully-featured to replace it.